lib/matplotlib/font_manager.py

```diff
@@ -1091,6 +1113,9 @@ def addfont(self, path):
         ----------
         path : str or path-like
         """
+        # Convert to string in case of a path as
+        # afmFontProperty and FT2Font expect this
+        path = str(path)
```

Comment:
I think this should use os.fsdecode instead? (to handle "arbitrary" pathlikes)

Reply:
Seems to be a better way indeed! Thanks!